src/styles/lite/buttons.scss

```diff
@@ -42,6 +49,22 @@
 
 }
 
+.note-icon-caret {
+  padding: 3px 5px 3px 0;
+}
+
+.note-icon-caret::before {
+  padding: 3px 2px 2px 0;
+}
+
+.note-btn .note-icon-caret:before {
+  padding: 3px 2px 3px 4px;
+}
+
+.note-btn.dropdown-toggle {
```

Comment:
Lite's dropdown button is like wrong style.

<img width="817" alt="스크린샷 2019-10-10 오후 2 30 35" src="https://user-images.githubusercontent.com/591983/66541538-91ff5000-eb6a-11e9-9711-b92d2b7c0869.png">

Reply:
Yes, I fixed that, that's what these changes do. Or am I not understanding what you mean?

Reply:
That padding fixes how they look so their heights are like the other buttons.